app/views/schools/advice/advice_base/insights.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= render 'schools/advice/advice_page', school: @school, advice_pages: @advice_pages, advice_page: @advice_page, tab: @tab do %>
+
+  <h2><%= t("advice_pages.#{@advice_page.key}.insights.title") %></h2>
+
+  <%= render 'insights' %>
+
+  <%= render 'schools/advice/section_title', section_id: 'recommendations', section_title: t('advice_pages.insights.recommendations.title') %>
+  <p><%= t("advice_pages.#{@advice_page.key}.insights.next_steps") %></p>
+
+  <%= render(RecommendationsComponent.new(recommendations: @activity_types, title: 'Work with the pupils', classes: 'pb-4')) %>
```

Comment:
do "Work with the pupils" & "Take action.." strings need to be extracted to translation files?